app/models/concerns/rbac.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+# Role-based Access Control
+module Rbac
+  extend ActiveSupport::Concern
+
+  attr_accessor :rbac_permissions
+
+  def permissions
+    return rbac_permissions if rbac_permissions.present?
+
+    self.rbac_permissions = request_access_data
+  end
```

Comment:
Memoization is much simpler than juggling with accessors and easier to read:
```suggestion
  def rbac_permissions
    @rbac_permissions ||= Rbac.load_users_permissions(account.b64_identity)
  end
```

Reply:
Also, could this be a private method?